demo/android/build.gradle

```diff
@@ -7,6 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:4.1.3'
         classpath "gradle.plugin.org.jsonschema2dataclass:jsonschema2dataclass:3.0.0"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
do we want introduce [platform](https://docs.gradle.org/current/userguide/java_platform_plugin.html) modules eventually?

Reply:
With a corresponding issue https://github.com/jitpack/jitpack.io/issues/3725

Reply:
Thanks, I'll think what I can do about